meta-resin-common/classes/image-resin.bbclass

```diff
@@ -193,6 +193,10 @@ resin_boot_dirgen_and_deploy () {
 
     echo "Install resin-boot in the rootfs..."
     cp -rvf ${RESIN_BOOT_WORKDIR} ${IMAGE_ROOTFS}/${RESIN_BOOT_FS_LABEL}
```

Comment:
Git log may use rephrasing

Reply:
I'll rephrase with next push. Thanks.

Reply:
Done

Reply:
NIT: the git log says: in the in boot partition
Could be fixed

Reply:
Fixed.